src/layouts/EnterpriseLoginLayout/EnterpriseLoginLayout.tsx

```diff
@@ -19,13 +20,17 @@ const FullPageLoader = () => (
 )
 
 function EnterpriseLoginLayout({ children }: { children: React.ReactNode }) {
+  const location = useLocation()
+
   const showExpiryBanner = localStorage.getItem(
     LOCAL_STORAGE_SESSION_EXPIRED_KEY
   )
   return (
     <>
       <Header />
-      {showExpiryBanner && <SessionExpiredBanner />}
+      {(location.search.includes('expired') || showExpiryBanner) && (
```

Comment:
I'd prefer to get the qparam by key and then check value. I don't see how this would happen but with this it's possible for `/login?branch=fix-session-expired` to trigger the banner. Just would rather keep it super explicit. If you disagree, I'll survive 😂

Reply:
lmfao that's a fair take and a good call out, tbh I'm too lazy to update all the tests and retest everything with the explicit query param 🙃

Reply:
+1 to that, we do have a query string package to help set and read query params.